tensorflow/compiler/tests/xla_call_module_test.py

```diff
@@ -396,8 +403,8 @@ def f(x):
       self._assertOpOutputMatchesExpected(f, (x,), (x,))
 
     # The module cannot have i64 %arg_platform_idx
-    module, version = serialize(module_str.replace('i32', 'i64'))
-    platforms = ['CPU', 'CUDA', 'TPU']
+    module = module.replace('i32', 'i64')
+    platforms = ['CPU', 'CUDA', 'ROCM', 'TPU']
```

Comment:
Is this `platforms` local variable used at all?

Reply:
Removed it.